### PR TITLE
CORE-6105 Remove avro dependency on net.corda.flow.rpcops

### PR DESCRIPTION
**Pull request by simon-johnson-r3**
- For reasons unknown at the moment, the avro base class org.apache.avro.specific.SpecificRecordBase cannot be located by the classloader in the combined worker when this dependency is added.
- Whilst this is a workaround, it's probably not such a bad idea not to expose avro so far up the public api of modules anyway.

### CODE REVIEW

File: libs/flows/utils/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ dependencies {
     implementation platform("net.corda:corda-api:$cordaApiVersion")
     implementation 'net.corda.kotlin:kotlin-stdlib-jdk8-osgi'
     implementation "net.corda:corda-base"
-    implementation "net.corda:corda-avro-schema"
+    api "net.corda:corda-avro-schema"
```
Comment (simon-johnson-r3) [inline libs/flows/utils/build.gradle:12]: This is not related to the workaround, although I noticed it was incorrect whilst looking into it (neither does it fix the problem in the combined worker).

File: libs/flows/utils/src/main/kotlin/net/corda/flow/utils/KeyValueStore.kt [hunk 2]
```diff
@@ -12,6 +12,12 @@ fun keyValueStoreOf(vararg pairs: Pair<String, String>) = KeyValueStore().apply
     }
 }
 
+fun keyValuePairListOf(map: Map<String, String>) = mutableKeyValuePairList().apply {
```
Comment (JamesHR3) [inline libs/flows/utils/src/main/kotlin/net/corda/flow/utils/KeyValueStore.kt:25]: Does this want a kdoc?

Comment (simon-johnson-r3) [reply libs/flows/utils/src/main/kotlin/net/corda/flow/utils/KeyValueStore.kt:25]: done

**Review by mbrkic-r3 — APPROVED** [review]
LGTM
